opal/opal_hot_reloader.rb

```diff
@@ -1,5 +1,7 @@
 require 'opal_hot_reloader/reactrb_patches'
 require 'opal-parser' # gives me 'eval', for hot-loading code
+require 'opal-jquery'
```

Comment:
I only use this for `Element.find` I think ... I'm not clear on if this comes with the standard opal setup or if this is an unnecessary dependency or what.